src/utils.rs

```diff
@@ -8,11 +8,7 @@ use std::{
 
 /// Are we running in in a CI environment?
 pub fn is_ci() -> bool {
-    match env::var("CI").ok().as_deref() {
-        Some("false") | Some("0") | Some("") => false,
-        None => env::var("TF_BUILD").is_ok(),
-        Some(_) => true,
-    }
+    env::var_os("CI").is_some() || env::var_os("TF_BUILD").is_some()
```

Comment:
What is the motivation of this change? This now means that it cannot be disabled any more by setting to an empty string.

Reply:
I think I stumbled upon this when I had to change the `cd` dance in the Makefile, but that's no longer correct until #410 lands. Reverted.